package.json

```diff
@@ -3,18 +3,16 @@
   "name": "root",
   "repository": "git@github.com:connectrpc/connect-query-es.git",
   "scripts": {
-    "build": "pnpm --recursive build",
-    "generate": "pnpm --recursive generate",
-    "test": "pnpm --recursive test",
-    "lint": "pnpm run lint:code && pnpm run lint:markdown && pnpm run lint:spelling",
-    "lint:fix": "pnpm run lint:code:fix && pnpm run lint:markdown:fix && pnpm run lint:spelling",
-    "lint:markdown": "markdownlint-cli2 \"**/*.md\" \"#**/node_modules\"",
-    "lint:markdown:fix": "markdownlint-cli2-fix \"**/*.md\" \"#**/node_modules\"",
-    "lint:spelling": "cspell \"**\" --gitignore",
-    "lint:code": "eslint . && prettier . --check && license-header --check",
-    "lint:code:fix": "eslint . --fix && prettier . --write && license-header",
-    "all": "pnpm run build && pnpm run generate && pnpm run lint:fix && pnpm run test",
-    "update-all": "pnpm outdated --recursive || pnpm update --recursive --latest && pnpm outdated --recursive"
+    "build": "turbo run build",
+    "generate": "turbo run generate",
+    "test": "turbo run test",
+    "format": "turbo run format:all",
```

Comment:
Not the end of the world, but `pnpm run format` fails when packages have not been built before. You can see it with 

```
git clean -Xdf
pnpm i
pnpm run format
```

I don't think `"dependsOn": ["^generate"]` works.

In my experience, turbo works best when each package uses it's own scripts. It's extra set-up, for sure.

Did you know you can run turbo in a package? For example, you can run `turbo test` in examples/react/basic, and never have to think about what to build first, and it will never do unnecessary work.

Reply:
I did not know turbo worked in a single package though, that is very cool.